exonum/src/events/codec.rs

```diff
@@ -21,10 +21,11 @@ use tokio_io::codec::{Decoder, Encoder};
 use messages::{HEADER_LENGTH, MessageBuffer, RawMessage};
 use super::error::other_error;
 
-pub const MAX_MESSAGE_LEN: usize = 1024 * 1024; // 1 MB
-
 #[derive(Debug)]
-pub struct MessagesCodec;
+pub struct MessagesCodec {
+    /// Maximum message length (in bytes), gets populated from `NetworkConfiguration`.
+    pub max_message_len: usize,
```

Comment:
Should this field be public?
I prefer to make a separate constructor and getter.

Reply:
As discussed, made it just private and initializable with constructor